plugins/services/src/js/reducers/JSONMultiContainerParser.js

```diff
@@ -23,6 +25,17 @@ module.exports = [
   simpleParser(["disk"]),
   simpleParser(["gpus"]),
   simpleParser(["cmd"]),
+  function scheduling(state) {
+    if (state == null) {
```

Comment:
:nail_care: these two conditions could be combined into `if (state == null || state.scheduling == null)`